Modifications in JPEG2000 module to output more info in mix formats

tledoux opened the pull request:
- Add CaptureResolution property in Jhove output
- Use either CaptureResolution or DefaultDisplayResolution in mix SpatialMetrics
- Handle lossy and lossless compression scheme in mix
- Calculate the compression ratio and output it in mix
- Add layers, levels, tile size in mix:SpecialFormatCharacteristics
- Fixes #106

tledoux committed:
Modifications in JPEG2000 module to output more info in mix formats
- Add CaptureResolution property in Jhove output
- Use either CaptureResolution or DefaultDisplayResolution in mix SpatialMetrics
- Handle lossy and lossless compression scheme in mix
- Calculate the compression ratio and output it in mix
- Add layers, levels, tile size in mix:SpecialFormatCharacteristics
- Fixes #106

tledoux committed:
Take care of codavy warnings

tledoux committed:
Remove unused imports

carlwilson committed:
Merge branch 'integration' into issue106

carlwilson reviewed (commented):
Hi @tledoux I think that the only task that remains is to bump the version number for the JPEG Module. I'm going to run some tests and make this change then merge the PR. It'll be done for w/e 10th May.

tledoux commented:
Be aware the version of the **JPEG2000** module should be bumped...

carlwilson committed:
TEST: Bumped versions and patched tests
- copied valid JP2K corpus example output due to new MIX metadata; and
- bumped version details for `JPEG2000-hul` and `XMLHandler`.

carlwilson committed:
FIX - patched up test script for new metadata.

tledoux committed:
Merge pull request #1 from carlwilson/issue106

FIX - Tests for JP2K MIX Metadata

carlwilson committed:
Merge branch 'integration' into issue106
